cdc/sink/manager_test.go

```diff
@@ -92,8 +92,14 @@ func (s *managerSuite) TestManagerRandom(c *check.C) {
 	var wg sync.WaitGroup
 	tableSinks := make([]Sink, goroutineNum)
 	for i := 0; i < goroutineNum; i++ {
-		tableSinks[i] = manager.CreateTableSink(model.TableID(i), 0)
+		i := i
```

Comment:
`i := i`， maybe` j :=i`

Reply:
The variable `i` is deliberately shadowed to avoid programmer mistakes

Reply:
Because of closure.? But with the same name kind of strange, imo

Reply:
```
It may seem odd to write

req := req

but it's legal and idiomatic in Go to do this. You get a fresh version of the variable with the same name, deliberately shadowing the loop variable locally but unique to each goroutine.
```

This way is widely used in Go, ref: https://golang.org/doc/effective_go